src/loginPage/LoginPage.js

```diff
@@ -1,15 +1,72 @@
-import React from 'react';
+import React, {useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+
+//TODO: replace to a separate file
+async function newSession(login, password) {
+    //TODO: replace to constants
+    const SERVER_PATH = 'http://localhost:5000';
+    const endPoint = SERVER_PATH + '/sessions/login';
+    let failedToFetch = false;
+
+    const response = await fetch(
+        endPoint,
+        { method: 'post',
+            body: JSON.stringify({ login, password }),
+            headers: { 'Content-type': 'application/json'}
+        })
+        .catch(() => failedToFetch = true);
```

Comment:
```suggestion
        .catch(() => {
          failedToFetch = true
        });
```

Reason: https://eslint.org/docs/latest/rules/no-return-assign